### DISCUSSION TITLE
How to deploy to specific region using CLI only

### DISCUSSION THREAD

**Discussion by jasper95** [vercel/community#2928] [Help] [answered]
### Summary

I made a script to deploy a nextjs project to vercel using CLI. The requirement is to deploy the functions to a specific region.
I followed the [docs](https://vercel.com/docs/cli/deploy#regions) about it but it didn't work for me.

### Steps to Reproduce

1. Link project
```
vercel link --yes --scope $ORG --token $TOKEN
```
2 Deploy to specific region
```
vercel --regions  sfo1 --token $TOKEN
```

**Comment by amyegan** [comment, marked as answer]
Hi @jasper95! You need to include the region, like `vercel --regions sfo1`

**Reply by jasper95** [reply]
sorry that was my bad on my post. I did use the correct parameter in my case
```
vercel --regions  sfo1 --token $TOKEN
```

but it still didn't work

**Reply by jasper95** [reply]
this is resolved. It actually worked. I was looking on the wrong place.
